main.go

```diff
@@ -24,6 +24,7 @@ func main() {
 	cfg, err := config.Get()
 	if err != nil {
 		log.Error(fmt.Errorf("error configuring service: %s. Exiting", err), nil)
+		os.Exit(1)
```

Comment:
Useful addition! Very confusing that bad config in Docker tilt shows up as green on exit (0).